scripts/reindex.py

```diff
@@ -121,6 +126,16 @@ def conda_data_wrapper(args):
 
 
 if __name__ == "__main__":
+    # the classifiers endpoint is populated by scripts/bigquery.py and it
+    # contains the list of active versions on PyPI with the napari classifier -
+    # we get the available versions from there to include in the summary
+    CLASSIFIERS = PUBLIC / "classifiers.json"
+    try:
+        active_pypi_versions = json.loads(CLASSIFIERS.read_text())["active"]
+    except Exception as e:
+        print(f"failed to retrieve active PyPI versions from  {CLASSIFIERS}")
```

Comment:
```suggestion
    except Exception:
        print(f"failed to retrieve active PyPI versions from  {CLASSIFIERS}", file=sys.stderr)
```

or print somehow the exception content